great_expectations/rule_based_profiler/parameter_builder/regex_pattern_string_parameter_builder.py

```diff
@@ -237,7 +237,17 @@ def _build_parameters(
             attributed_resolved_metrics
         ) in metric_computation_result.attributed_resolved_metrics:
             # Now obtain 1-dimensional vector of values of computed metric (each element corresponds to a Batch ID).
-            metric_values = attributed_resolved_metrics.conditioned_metric_values[:, 0]
+            metric_values: MetricValues
+
+            metric_values = attributed_resolved_metrics.conditioned_metric_values
+
+            if metric_values is None:
+                raise gx_exceptions.ProfilerExecutionError(
+                    message=f"Result of metric computations for {self.__class__.__name__} is empty."
+                )
```

Comment:
Could this also be a empty list/dict etc?
```suggestion
            if not metric_values:
                raise gx_exceptions.ProfilerExecutionError(
                    message=f"Result of metric computations for {self.__class__.__name__} is empty."
                )
```